File: 백준/Bronze/2525. 오븐 시계/오븐 시계.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');
const [A, B] = input[0].split(' ').map(Number);
const C = +input[1];
let [fA, fB] = [A, B + C];

if (fB > 59) {
    fA += Math.floor(fB / 60);
    fB %= 60;
}

if (fA > 23) fA -= 24;

console.log(fA, fB);

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
H, M = map(int, input().split())
T = int(input())
S = M + T

if S >= 60:
    H = int((H + (S / 60)) % 24)
    M = (M + T) % 60
else:
    M = S
    
print(H, M)